examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -98,7 +98,7 @@ class WebGPURenderer {
 		this.autoClearDepth = true;
 		this.autoClearStencil = true;
 
-		this.outputColorSpace = LinearSRGBColorSpace;
+		this.outputColorSpace = SRGBColorSpace;
```

Comment:
Oh! I almost forgot...

I think this should be:

```js
this.outputColorSpace = null;
```

This will allow the renderer to use `SRGBColorSpace` or `DisplayP3ColorSpace` automatically based on the user's display.

Reply:
Is it already clear how the renderer will determine the user's display color space? I suppose there will be a special Web API for this?

Reply:
I think we can do this check: https://github.com/mrdoob/three.js/issues/23614#issuecomment-1434281946

Reply:
By setting it to `null` it will also leave the door open for `HDRColorSpace` or whatever we get in the future.

Reply:
Hmm...

I'm searching the internal usage of `outputColorSpace` in `/src` and setting in to `null` could indeed complicate things a bit.

The constructor could just do this too:

```js
const isP3Display = window.matchMedia( '(color-gamut: p3)' ).matches;

this.outputColorSpace = isP3Display ? DisplayP3ColorSpace : SRGBColorSpace;
```

I think that'd be the first parameter that we have that is dynamic like that but I think I'm open to it... 🤔

Reply:
To me, that would be the most intuitive solution. I think it's understandable for users why the output color space is a) determined by the renderer and b) might vary from display to display.

Reply:
Alright. Lets do that 👍

Reply:
BTW: `DCI-P3` related features are tracked by the task 2.2 in #23614.

Looking at the spec, I wonder if defining `DisplayP3ColorSpace` is sufficient. Is it required to change the `WebGLRenderingContext.drawingBufferColorSpace`, too? It accepts `srgb` and `display-p3` with `srgb` as the default.

When I change this property in any `three.js` example to `display-p3`, I can actually see a hue shift.

Reply:
Let's enable color management by default first before going for the `DisplayP3ColorSpace` change. We need the color space conversion in any event to get the colors in `DisplayP3ColorSpace` right. 

https://github.com/mrdoob/three.js/issues/23614#issuecomment-1500212921

Reply:
Even if `encodings_fragment` were implemented for Display P3, I'm not sure yet when it should be turned on. The difference when enabling it for existing content is very subtle — not an automatic improvement.

There are more impactful results when bringing in wide-gamut colors and textures in scenes designed for Display P3, particularly for unlit scenes. Or perhaps for rendering lit scenes in a wide-gamut working color space. But either of those will require more investigation.

For now I think we should wait — SRGBColorSpace will be the right default for a while yet.